config: iam create aws check zone contains availability zone (#1913)
edgelesssys/constellation: docs/docs/workflows/config.md

## Creating an IAM configuration

You can create an IAM configuration for your cluster automatically using the `constellation iam create` command.

> If you already have a constellation configuration file, you can add the `--update-config` flag to the command. This writes the needed IAM fields into your configuration. Furthermore, the flag updates the zone/region of the configuration when it's not yet set.

Reviewer: ```suggestion
If you already have a Constellation configuration file, you can add the `--update-config` flag to the command. This writes the needed IAM fields into your configuration. Furthermore, the flag updates the zone/region of the configuration if it hasn't been set yet.
```
nit: grammar
(and I also missed the lower case Constellation in the other PR)

> If you already have a constellation configuration file, you can add the `--update-config` flag to the command. This writes the needed IAM fields into your configuration. Furthermore, the flag updates the zone/region of the configuration when it's not yet set.

Reviewer: ```suggestion
If you already have a Constellation configuration file, you can add the `--update-config` flag to the command. This writes the needed IAM fields into your configuration. Furthermore, the flag updates the zone/region of the configuration if it hasn't been set yet.
```
this got lost somehow
